File: lib/finger_model.dart
class FingerModel {
  final String taskId;
  final String key;
  final String baseUrl;
  final String token;

  FingerModel({
    required this.taskId,
    required this.key,
    required this.baseUrl,
    required this.token,
  });

  factory FingerModel.fromJson(Map<String, dynamic> json) {
    return FingerModel(
      taskId: json['task_id'],
      key: json['key'],
      baseUrl: json['base_url'],
      token: json['token'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'task_id': taskId,
      'key': key,
      'base_url': baseUrl,
      'token': token,
    };
  }
}

File: android/src/main/kotlin/uz/udevs/finger_print_pad/model/FingerModel.kt
package uz.udevs.finger_print_pad.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class FingerModel(
    @SerializedName("task_id")
    val taskId: String,
    @SerializedName("token")
    val token: String,
    @SerializedName("key")
    val key: String,
    @SerializedName("base_url")
    val baseUrl: String,
) : Serializable